bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCheckIssueVisitor.java

```diff
@@ -72,6 +72,14 @@ List<String> errorFailedChecksMessages() {
         return errorFailedChecks.values().stream().flatMap(List::stream).toList();
     }
 
+    List<String> errorFailedCheckMessagesWithoutReviewersCheck() {
```

Comment:
This is a mouthful. I was thinking something like `boolean tooFewReviewers()`.

Reply:
Do you mean that `tooFewReviewers()` should return true when there is only ReviewersCheck Failure?

Reply:
Or does `tooFewReviewers()` return true when there is a ReviewersCheck Failure in the list?
If so, the logic should be  like this
`            var readyToPostApprovalNeededComment = readyForReview &&
                    ((!reviewNeeded && visitor.errorFailedChecksMessages().size() == 1 && visitor.tooFewReviewers()) ||
                            visitor.errorFailedChecksMessages().isEmpty()) &&
                    integrationBlockers.isEmpty() &&
                    !statusMessage.contains(TEMPORARY_ISSUE_FAILURE_MARKER);`

Reply:
Right, I didn't think that through and this gets messy and I think it's because we aren't using the right abstraction. Can we just push something like this down to the visitor:
```
boolean hasErrors(boolean reviewNeeded)
```

Reply:
Yes, sounds good